semgrep/semgrep/util.py

```diff
@@ -20,6 +21,10 @@
 
 T = TypeVar("T")
 
+YML_EXTENSIONS = {".yml", ".yaml"}
+YML_SUFFIXES = [[ext] for ext in YML_EXTENSIONS]
+YML_TEST_SUFFIXES = [[".test", ext] for ext in YML_EXTENSIONS]
```

Comment:
Wondering if we should be using soemthing like .r2ctest or something specific in case a user actually has .test.yaml?

Reply:
Hmm, we could do that, but we've already released a version using `.test`, and all our `semgrep-rules` use `.test` (not that that's unchangeable). I'm less worried about YAML test files (`.test.yaml`) because that tends to be a much more tightly controlled directory structure, i.e. it's typically just Semgrep rules and associated tests. The config file suffix comparison should be much stricter because it runs across diverse directories, i.e. arbitrary user repositories.